gensim/models/keyedvectors.py

```diff
@@ -278,11 +278,12 @@ def word_vec(self, word, use_norm=False):
         """
         if word in self.vocab:
             if use_norm:
-                return self.syn0norm[self.vocab[word].index]
+                result = self.syn0norm[self.vocab[word].index]
             else:
-                return self.syn0[self.vocab[word].index]
-        else:
```

Comment:
Why you remove this branch? We must raise an exception if word not in the dictionary.